### PR TITLE
[WIP] Move install/authenticate instructions from CLI reference to install

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

The CLI Reference section begins with instructions to install and authenticate the CLI, which is also discussed in the main Installation section.  This PR merges the info from CLI ref into the appropriate sections in the installation docs.

### CODE REVIEW

File: content/docs/1.0.x/reference/cli/_index.md [hunk 1]
```diff
@@ -1,197 +1,33 @@
...
-```console
-keptn version
-```
+This section provides detailed usage information for the Keptn CLI commands.
+The Keptn CLI includes commands to perform various tasks
+to manage the Keptn installaton and the projects that are run.
+It is packaged separately from the Keptn product itself.
+It is possible to run current versions of Keptn without the Keptn CLI;
+most basic functionality can instead be performed
+using the [Keptn bridge](../../bridge).
```
Comment (aepfli) [inline content/docs/1.0.x/reference/cli/_index.md:14]:
```suggestion
using the [Keptn bridge](../../bridge/).
```

File: content/docs/1.0.x/reference/cli/_index.md [hunk 2]
```diff
@@ -1,197 +1,33 @@
...
+However, it is recommended that the CLI be installed for production environments
+because some functionality is available only through the CLI.
+
+See [Install Keptn CLI](../../install/cli-install) for full installation instructions.
```
Comment (aepfli) [inline content/docs/1.0.x/reference/cli/_index.md:18]:
```suggestion
See [Install Keptn CLI](../../install/cli-install/) for full installation instructions.
```

File: content/docs/1.0.x/reference/cli/_index.md [hunk 3]
```diff
@@ -1,197 +1,33 @@
...
+After you install both the Keptn CLI and Keptn itself,
+you must authenticate the CLI and the Bridge
+as discussed in
+[Authenticate Keptn CLI and Bridge](../../install/authenticate-cli-bridge).
```
Comment (aepfli) [inline content/docs/1.0.x/reference/cli/_index.md:22]:
```suggestion
[Authenticate Keptn CLI and Bridge](../../install/authenticate-cli-bridge/).
```

**Review by aepfli — APPROVED** [review]
lgtm - just 3 trailing slashes for URLs are missing, due to our latest SEO improvements

**Comment by StackScribe** [comment]
This PR is superceeded by https://github.com/keptn/keptn.github.io/pull/1567 .

**Pull request closed without merging**